### ISSUE TITLE
Why val_acc is bigger than train_acc?

### ISSUE THREAD

**Issue by JackbtSun** [pss1207/sleep_apnea#10]
Like the question, do you have some problems with the division of the data set? The accuracy of the training set is about 85 percent, while the validation set actually reaches 95 percent
